packages/teams-js/test/public/sharing.spec.ts

```diff
@@ -23,12 +31,335 @@ describe('sharing_v1', () => {
     }
   });
 
-  it('should successfully call the callback function when given the share web content in correct format - success scenario', done => {
-    utils.initializeWithContext('content').then(() => {
-      // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-      const callback = () => {
-        done();
-      };
+  describe('Testing sharing.shareWebContent function v1', () => {
+    Object.keys(FrameContexts)
+      .map(key => FrameContexts[key])
+      .forEach(frameContext => {
+        if (!allowedContexts.includes(frameContext)) {
+          it(`sharing.shareWebContent should not allow calls from ${frameContext} context`, async () => {
+            await utils.initializeWithContext(frameContext);
+            const shareRequest: sharing.IShareRequest<sharing.IURLContent> = {
+              content: [
+                {
+                  type: 'URL',
+                  url: 'https://www.microsoft.com',
+                  preview: true,
+                  message: 'Test',
+                },
+              ],
+            };
+            expect(() => sharing.shareWebContent(shareRequest)).toThrowError(
+              `This call is only allowed in following contexts: ${JSON.stringify(
+                allowedContexts,
+              )}. Current context: "${frameContext}".`,
+            );
+          });
+        }
+      });
+
+    it('sharing.shareWebContent should successfully call the callback function when given the share web content in correct format - success scenario', done => {
```

Comment:
It looks like we only test the `sharing.shareWebContent` function success scenario with FrameContexts.content. Since we allow content, sidePanel, task, stage, and meetingStage, can you add the success scenario with those contexts as well please?

Reply:
I have added it for both v1 and v2 please check the latest commit.